test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -191,78 +200,100 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
                             tagHelper.FallbackTestClass = "hidden";
                             tagHelper.FallbackTestProperty = "visibility";
                             tagHelper.FallbackTestValue = "hidden";
-                        }
-                    },
-                    // File Version
-                    {
-                        new TagHelperAttributeList
-                        {
-                            ["asp-append-version"] = "true"
-                        },
-                        tagHelper =>
-                        {
                             tagHelper.AppendVersion = true;
                         }
```

Comment:
Add a new test that just sets `AppendVersion`. `HrefInclude` is now set in every case that includes `AppendVersion`.